inference-engine/ie_bridges/python/CMakeLists.txt

```diff
@@ -53,6 +53,7 @@ endif()
 
 set (PYTHON_BRIDGE_SRC_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 add_subdirectory (src/openvino/inference_engine)
+add_subdirectory (src/openvino/offline_api)
```

Comment:
maybe `moc_api`?

Reply:
renamed to offline_transformatoins. Library was renamed to offline_transformations_api.